eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -43,6 +43,8 @@ class Peer(nodeParams: NodeParams, remoteNodeId: PublicKey, authenticator: Actor
 
   import Peer._
 
+  implicit val ec: ExecutionContext = context.system.dispatcher
```

Comment:
I'd rather put that kind of things exactly were it is needed, otherwise we are never quite sure what part of the code relies on it and it can lead to tricky debugging.